### DISCUSSION TITLE
Mussels & Contaminants

### DISCUSSION THREAD

**Discussion by ChrisMantegna** [RobertsLab/resources#1407] [Ideas]
Do we have any connections to the Mussel Watch program? I'm broadly thinking of two reasons: (1) accessing their historical contamination data that I cannot find in anything more than public summary, and / or (2) who processes all of their samples, are any stored, could we gain access to this information? It is coordinated by WDFW and NOAA, but it looks like local non-profits do the data collection work, mainly Puget Soundkeepers from what I can see. 
Thoughts? @sr320 @kubu4 @mattgeorgephd 
Thanks y'all!

**Comment by mattgeorgephd** [comment]
Not sure who is heading it now. Jennifer Lanksbury recently moved to a position at King County. I've reached out to a few people - I'll let you know what I find out.

Relevant info:

[Quality assurance plan](https://wdfw.wa.gov/sites/default/files/publications/01596/wdfw01596.pdf)

[WDFW 2014 publication](https://wdfw.wa.gov/sites/default/files/publications/01643/wdfw01643.pdf)

[James et al 2020 - uses data](https://www.sciencedirect.com/science/article/pii/S0048969719360942?casa_token=buiI_GgDnegAAAAA:Lt7KGoQnEZj-5-XCE0ftQXIg-Qyt4en-YyOGnXH4KPBl9D7d5VaS3nT0Ziw4eezN5aeWndhM#bb0255)
